apps/wallet/src/ui/app/experimentation/feature-gating.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GrowthBook } from '@growthbook/growthbook';
+
+export default class FeatureGating {
+    private _growthBook: GrowthBook;
```

Comment:
Use `#growthBook` instead of TS privates.

Reply:
done!